packages/@tinacms/cli/src/cmds/init/index.ts

```diff
@@ -93,12 +87,12 @@ export async function tinaSetup(ctx: any, next: () => void, options) {
     fs.writeFileSync(blogPostPath, blogPost)
   }
 
-  // 2. Create a Tina Wrapper
-  if (!fs.pathExistsSync(TinaWrapperPath)) {
-    logger.info(logText('Adding a tina-wrapper...'))
-    fs.mkdirpSync(TinaWrapperPathDir)
-    fs.writeFileSync(TinaWrapperPath, TinaWrapper)
-  }
+  // // 2. Create a Tina Wrapper
```

Comment:
@logan-anderson I don't think we need a wrapper of any kind anymore, any reason you can think of?

Reply:
If a components/tina-wrapper.tsx is not needed then this is not needed. So yes